docs/admin/provisioning_usb.rst

```diff
@@ -38,5 +39,45 @@ of its airgap:
   again. The formatting process should take only a few seconds.
 - Once formatting is complete, you will need to provide the *Export Device* and
   its decryption password to the SecureDrop Workstation users. Make sure that
-  they store it and its password securely, as it will contain decrypted 
+  they store it and its password securely, as it will contain decrypted
+  submissions.
+
+Creating a VeraCrypt-encrypted drive
+------------------------------------
+
+- If it isn't already done, download and install the `VeraCrypt software <https://www.veracrypt.fr/en/Home.html>`_.
+- Start VeraCrypt from your computer's application or software interface.
+- Click **Create Volume**
+- Select **Encrypt a non-system partition/drive** and click **Next**.
+- Select **Standard VeraCrypt volume** and click **Next**
+- Connect your fresh USB stick and click **Select Device...** to choose your USB.
+
+  - You may see a warning that says "We strongly recommend that inexperienced
```

Comment:
Do we want people to only encrypt partitions? Means we need to guide them into creating a partition first I think

Reply:
They can encrypted a whole device or a partition, either way

Reply:
Oh good point, I've added something like "We disagree with this recommendation..." just to make it clearer it can be ignored.